core/src/main/java/hudson/slaves/JNLPLauncher.java

```diff
@@ -76,6 +77,13 @@
 
     private boolean webSocket;
 
+    /**
+     * @see #getInboundAgentUrl()
+     */
+    @NonNull
+    @Restricted(NoExternalUse.class)
+    public static final String CUSTOM_INBOUND_URL_PROPERTY = "jenkins.agent.inboundUrl";
```

Comment:
Why is this format different from basically all other system properties in Jenkins?